src/app/views/query-runner/request/feedback/campaignDefinitions.ts

```diff
@@ -0,0 +1,82 @@
+const CampaignDefinitions: object[] = [{
+  'CampaignId': getCampaignId().toString(),
+  'StartTimeUtc': '2017-01-01T00:00:00Z',
+  'EndTimeUtc': null,
+  'GovernedChannelType': 0,
+  'AdditionalDataRequested': ['EmailAddress'],
+  'Scope':
+    {
+      'Type': 0
+    },
+  'NominationScheme':
+    {
+      'Type': 0,
+      'PercentageNumerator': 100,
+      'PercentageDenominator': 100,
+      'NominationPeriod':
+        {
+          'Type': 0,
+          'IntervalSeconds': 120
```

Comment:
The interval can be different for different environments. If in development, the seconds can be few and in production they can be alot.
Also in production, possibly putting the seconds in an environment variable would be helpful for instances where we'd like to change the interval time.

`IntervalSeconds: (NODE_ENV === 'development') ? 120 : process.ENV.IntervalAtThisPointWithRelevantName`